mmv1/templates/terraform/examples/go/test_file.go.tmpl

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
I wonder if this file can be moved to `mmv1/templates/terraform` folder, which has resource and doc templates?

Reply:
I moved it to `mmv1/templates/terraform/examples/base_configs/` since that is where the `test_file.go.erb` is right now